bower.json

```diff
@@ -22,7 +22,14 @@
     "angular-sanitize": "^1.5.5",
     "firebase": "^2.4.2",
     "angularfire": "^1.2.0",
-    "isotope": "1.5.21",
-    "imagesloaded": "^4.1.0"
+    "imagesloaded": "^4.1.0",
+    "isotope": "^1.5.21"
+  },
+  "overrides": {
+    "font-awesome": {
+      "main": [
+        "css/font-awesome.css"
+      ]
+    }
   }
```

Comment:
I think it works if we use the most recent version of isotope but when I tried to update it the view which uses it broke completely, so it would have to be rewritten. Did you know that we are using an old version of isotope?